woocommerce/woocommerce
# Add `Cardo` and `Inter` fonts to global styles via theme json filter

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woocommerce/issues/47441

Addresses p1715591224614979/1715346843.941069-slack-C01SFMVEYAK

This PR experiments adding `Cardo` and `Inter` fonts to global styles via `wp_theme_json_data_theme`, which originally came from TwentyTwentyFour theme. This allows for WooCommerce plugin to use these fonts without the original theme.

This PR is still an experiment due to:

1. We should think of a better structure to use `wp_theme_json_data_theme` in WooCommerce, since this piece of code affects the entire site.  

Also, follow-up work will be needed to set the fonts in coming soon pages.

### How to test the changes in this Pull Request:

1. Install and activate the [FSE Empire](https://wordpress.org/themes/fse-empire/) theme
1. Go to `WooCommerce > Settings > Site visibility`
3. Ensure `Coming soon` is selected and `Restrict to store pages only` is disabled
4. Go to `Appearance > Editor > Templates > Page: Coming soon` and edit
5. Select the center heading and open up `Styles > Typography`
6. Observe `Cardo` and `Inter` fonts are listed in the option

<img width="1253" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/244d837e-b9e0-4976-8088-6cf196c6c77a">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
